Packages/Core/Runtime/Variables/AtomVariable.cs

```diff
@@ -1,6 +1,7 @@
 using System;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEditor;
```

Comment:
needs to be wrapped in `#if UNITY_EDITOR` or the unity project will not build

Reply:
Have been addressed in the second commit